onvm/lib/cJSON.c

```diff
@@ -633,14 +633,17 @@ static const char *parse_string(cJSON *item, const char *str, const char **ep)
                             /* 10xxxxxx */
                             *--ptr2 = ((uc | 0x80) & 0xBF);
                             uc >>= 6;
+                            break;
```

Comment:
This technically changes the behavior, instead of falling through all the cases it only does the one.  

Although I'm not sure what the expected behavior is anyway. Should we just get a new cJSON version?

Reply:
is there any need to fall-through all of the cases? it switches on len, so only one of those cases is going to be hit. why the need to continue through the switch statement if only case 4,3,2 or 1 is hit?